skyflow/vault-api/detokenize.go

```diff
@@ -169,3 +173,11 @@ func (detokenize *DetokenizeApi) sendRequest(records common.DetokenizeInput) (ma
 	}
 	return finalRecord, nil
 }
+
+func appendRequestId(message string, requestId string) string {
```

Comment:
why is this in detokenize, its common for all interfaces right ?

Reply:
moved it to helpers file